site/_config.yml

```diff
@@ -33,6 +33,12 @@ exclude:
   - Gemfile
   - Gemfile.lock
   - vendor
+  - init.go
+  - app.yaml
+  - package.json
+  - service-acc.json.gpg
+  - sw-precache-config.js
+  - node_modules
```

Comment:
Yes good catch! Was planning to remove it after I merge this but I'll appreciate if you remove it in a commit :) It was there because we used an app-engine plugin initially but we're not using it any more.

Thanks a million for all the help cleaning things up with the setup, we appreciate it!